torch/csrc/jit/codegen/cuda/parser.h

```diff
@@ -26,6 +26,15 @@ namespace jit {
 namespace fuser {
 namespace cuda {
 
+#define PW_THREAD_X 128
```

Comment:
constexpr int PW_THREAD_X = 128; ...

NOTE: we should probably follow suit with the rest of PyTorch C++ code and use the kFoo naming for compile time constants (ex. kPwThreadX instead of PW_THREAD_X). I'll leave it up to you if you want to make these changes, or we can do a mass rename pass later.

Reply:
Updated. Feel free to yell at me if you notice I missed anything :)